code/modules/awaymissions/mission_code/ruins/oldstation.dm

```diff
@@ -498,3 +479,58 @@
 	name = "Hivebot Mothership"
 	icon_state = "teleporter"
 	requires_power = FALSE
+
+// MARK: LORE CONSOLES
+
+/obj/machinery/computer/loreconsole/oldstation/damage_report
+	entries = list(
+		new/datum/lore_console_entry(
+			"Damage Report",
+			{"**Alpha Station** - Destroyed.
+
+**Beta Station** - Catastrophic Damage. Medical, destroyed. Atmospherics, partially destroyed. Engine Core, destroyed.
+
+**Charlie Station** - Intact. Loss of oxygen to eastern side of main corridor.
+
+**Theta Station** - Intact. **WARNING**: Unknown force occupying Theta Station. Intent unknown. Species unknown. Numbers unknown.
+
+Recommendation - Reestablish station powernet via solar array. Reestablish station atmospherics system to restore air."}))
+
+/obj/machinery/computer/loreconsole/oldstation/crew_awakening_report
+	entries = list(
+		new/datum/lore_console_entry(
+			"Crew Awakening Report",
+			{"## WARNING
+
+Catastrophic damage sustained to station.
+Powernet exhausted to reawaken crew.
+
+## Immediate Objectives
+
+1. Activate emergency power generator.
+2. Lift station lockdown on the bridge.
+
+Please locate the 'Damage Report' on the bridge for a detailed situation report."}))
+
+/obj/machinery/computer/loreconsole/oldstation/report
+	name = "\improper Crew Reawakening Report"
+
+/obj/machinery/computer/loreconsole/oldstation/report/Initialize(mapload)
+	. = ..()
+	init_current_date_string()
+	entries += new/datum/lore_console_entry(
```

Comment:
IMO, it would be best to have these be subtypes when possible if we're going to datumize them. Otherwise they may as well be lists